Map/MapManager.gd

```diff
@@ -6,14 +6,19 @@ static func create_map(): ## Generates and Populates a map with rooms that have
 	var _grid: Array = [] ## 2d array to return. this will be populated with rooms
 	
 	for index_height: int in range(_map.floors_width.size()): ## loop through elements of the grid and assign a room type
+		var _max_floor_size = _map.floors_width[_map.floors_width.size()/2]
 		_grid.append([])
+		var _padding = (_max_floor_size - _map.floors_width[index_height])/2
+		var _floor_width = _map.floors_width[index_height]
```

Comment:
if you put this before the previous line, you can more easily write previous line as:
`var _padding = (_max_floor_size - _floor_width)/2`